site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -25,6 +25,6 @@ Note: ZAP's home is not the same as the user's home directory, which is also OS
 
 `~/Library/Application Support/ZAP`
 
-##  Weekly releases
+##  Weekly releases and development builds
 
-Note that for weekly releases `_D` is appended to the above directory. Development builds also use this directory.
+Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the zap.log file will be saved in `~/.ZAP_D/`.
```

Comment:
```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the home directory is `~/.ZAP_D/`.
```

Reply:
Per the latest messages yesterday.

Reply:
+1 if the user uses the `-dir` option then that will be used for the log files instead of the home dir...